src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -3223,6 +3228,9 @@ NetworkOPsImp::pubAccountTransaction(
             RPC::insertDeliveredAmount(jvObj[jss::meta], *ledger, stTxn, meta);
         }
 
+        if (last)
+            jvObj[jss::account_history_ledger_boundary] = true;
```

Comment:
`jvObj[jss::account_history_ledger_boundary] = true;` should only be added if this is an account history subscribe. So these two lines should be moved into the for loop below.

Reply:
> jvObj[jss::account_history_ledger_boundary] = true; should only be added if this is an account history subscribe

Hum, I don't think so, the boundary information is useful in both directions (going back for historical txs and forward for new tx). Maybe I should rename `jss::account_history_ledger_boundary` to `jss::ledger_boundary`?

Reply:
The ledger boundary is needed for an optimization where we are submitting attestations in batch.

Reply:
So, since attestations are no longer submitted as a batch, the optimization is no longer required?

Reply:
It's not required, although I'd like to keep it. I do intend on re-add batch attestations, and this seems like a low-risk, minor addition to the rpc command. However, I also understand the desire to remove it. If you feel strongly, we can remove it, although I really would like to keep it.

Reply:
I don't feel strongly about it.  But XChain is a _big_ change.  I think it's smart to limit the scope where we can.  It sounds like this change can be postponed until batch attestations are re-added?  Is there anything critical about introducing this change _now_?  That said, if you need the change then you need it.

Also, this is the first time I've heard that you intend to re-add attestation batching.  Please think long and hard before re-adding batch attestations.  Batching attestations created several complexities:

- The batched handling inside the server was messy,
- The handling of mixed successful and unsuccessful attestations and their error codes,
- Multiple payments in a single transaction if attestations to different destinations are in the same batch.

I really doubt the benefits of batching out weigh the added complexity and opportunity for error.

Reply:
To be clear, I don't intend to add batch attestations before launch. They are an optimization. We can decide if we need them later.